src/extension/wps/wps-core/src/test/java/org/geoserver/wps/my-secret.xml

```diff
@@ -0,0 +1 @@
+<foo>MY_SECRET</foo>
```

Comment:
If you are going to make other changes to this PR, can you add the newline at the end of this file too?

Reply:
I added the newline at the end of this file.